cloudferrylib/scheduler/scheduler.py

```diff
@@ -21,13 +21,20 @@
 import signal_handler
 from task import BaseTask
 from thread_tasks import WrapThreadTask
-
+import oslo.config.cfg
 
 LOG = utils.get_log(__name__)
 
 
 NO_ERROR = 0
 ERROR = 255
+ERROR_INITIAL_CHECK = 1
+ERROR_INVALID_CONFIGURATION = 2
+ERROR_MIGRATION_FAILED = 3
+ERROR_DURING_ROLLBACK = 4
```

Comment:
Please move error codes to a separate file, for example to `cloudferrylib/utils/errorcodes.py`, so that they can be imported easily from other modules.